### DISCUSSION TITLE
Is birdseye latency affected by any changes in 0.12.0-beta1?

### DISCUSSION THREAD

**Discussion by thinkloop** [blakeblackshear/frigate#4909] [Ask A Question] [answered]
This question may show my ignorance, but does webrtc, for example, affect birdseye latency at all? I would like to be able to catch the uber person before they ring 😄

**Comment by NickM-27** [comment, marked as answer]
The delay in birdseye is from the detection and decoding, these have not been changed at all in 0.12
